TACC/protx-dashboard
# task/COOKS-325: display the dfps_region when mousing over the region

## Overview: ##

Display the dfps_region when mousing over the region

## Related Jira tickets: ##

* [COOKS-325](https://jira.tacc.utexas.edu/browse/COOKS-325)

## Summary of Changes: ##

## Testing Steps: ##
1. Go to [demograhpics](https://cep.test/protx/dash/demographics)
2. View the dfps regions
3. Ensure that when your mouse is over a region that you see the name of the region

## UI Photos:
![Screen Shot 2022-10-12 at 1 38 18 PM](https://user-images.githubusercontent.com/8287580/195422027-354c4a4c-6ca1-4aca-baaf-baa9d3ab7480.png)
